core-bundle/src/Security/Voter/BackendAccessVoter.php

```diff
@@ -26,12 +36,72 @@ protected function supports($attribute, $subject): bool
     protected function voteOnAttribute($attribute, $subject, TokenInterface $token): bool
     {
         $user = $token->getUser();
-        [, $field] = explode('.', $attribute, 2);
 
-        if (!$user instanceof BackendUser || (!is_scalar($subject) && !\is_array($subject))) {
```

Comment:
It seems that you have moved the `if (!$user instanceof BackendUser)` check down to line 52, however, I think it should be executed right here, because if we return early, we do not need to explode the attributes.

Reply:
Changed in 59532b99e4bc9016ad698f61898611c65263960d